examples/covid/canada.case-details.yaml

```diff
@@ -24,6 +24,19 @@ content:
               id,
               reported_date,
               gender,
+              case when age_group = 'Under 1 year' then '<20'
```

Comment:
Can you please move these transformations into `*.hm` datasets?

The idea of this pipeline is that:
- `<province>.case-details` root datasets contain **raw** data (mostly unaltered from the source)
- `<province>.case-details.hm` derivative datasets **harmonize** data of each province into a similar format (so that one could work with one province and then another without worrying about semantic differences between them)
- `canada.case-details` derivative dataset **combines already harmonized** data from all provinces into one dataset

So I would much prefer that `canada.case-details` contained no complex logic other than `UNION ALL` operations.

Reply:
Did what you said